src/config.ts

```diff
@@ -203,6 +203,16 @@ const StoryLikeSchema = z.object({
   parameters: z.record(z.unknown()).optional(),
 });
 
+const ShotItemSchema = z.object({
```

Comment:
Better define what is this, as there are more props that need to be defined here too